src/platforms/native/guides/minidumps/index.mdx

```diff
@@ -10,15 +10,12 @@ description: "Learn about how Sentry processes Minidump crash reports."
 Sentry can process Minidump crash reports, a memory dump used on Windows and by
 open-source libraries like [_Breakpad_](/platforms/native/guides/breakpad/) or [_Crashpad_](/platforms/native/guides/crashpad/).
 
-<Note>
 
-We strongly encourage you to use a higher-level SDK if possible, as Minidumps uploads are very limited in functionality:
+<Alert level="warning">
 
-- [_Native (C/C++)_](/platforms/native/)
-- [_Apple (Cocoa)_](/platforms/apple/)
-- [_Electron_](/platforms/electron/)
+This page describes **standalone** usage of Minidumps with Sentry. We strongly encourage you to use our [Native SDK](/platforms/native/), as Minidumps uploads are very limited in functionality.
```

Comment:
Why would we no longer list the Apple and Electron SDKs here?

Reply:
@torarnv Could you elaborate why Apple and Electron are no longer mentioned?

Reply:
I don't know if my comment wasn't posted or not. Does https://github.com/getsentry/sentry-docs/pull/2842#discussion_r553304455 clarify?